### ISSUE TITLE
Make dialog closable via escape key.

### ISSUE THREAD

**Issue by nikku** [nikku/jquery-bootstrap-scripting#11]
(When configured).

**Comment by andreslucena** [comment]
Hacked a little bit and I have this:

``` javascript
$(function() {
  ///////////////////////// dialog2 - close modal windows with ESC key - start
  function __getOverlay(dialog) {
      return dialog.parent().prev(".modal-backdrop");
  }

  $(".modal").bind("keydown" ,function(event) {
    if (event.keyCode == "27") {
      var dialog = $(this).children(".modal-body");
      __getOverlay(dialog).hide();

      dialog
          .parent().hide().end()
          .trigger("dialog2.closed")
          .removeClass("opened");
    }
  });
  ///////////////////////// dialog2 - close modal windows with ESC key - end

});
```

I don't know yet where and how to put it on your script, but I hope this help with the basic stuff (capturing event.keyCode = "27").

**Comment by nikku** [comment]
Fixed in `1.2-wip` branch.

Closing dialogues on escape is active as the default but may be changed using the `closeOnEscape: false` option.
